src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery(new Ball(),
+                new ColorSupplier(),
+                new NumberBall());
+        for (int i = 0; i < 3; i++) {
+            System.out.println(lottery.getRandomBall().toString());
```

Comment:
There is no need to call `toString()` explicitly when printing an object with `System.out.println`, as it implicitly calls the `toString()` method of the object. You can simplify this to `System.out.println(lottery.getRandomBall());`

Reply:
Removed toString()

Reply:
![image](https://github.com/user-attachments/assets/02926cd4-9735-4c78-b180-bbf178a31e68)
you don`t removed